docs/manual/install/index.rst

```diff
@@ -111,3 +111,32 @@ Stable versions of Qtile can be installed from PyPI:
 As long as the necessary libraries are in place, this can be done at any point,
 however, it is recommended that you first install xcffib to ensure the
 cairo-xcb bindings are built (see above).
+
+Wayland
+-------
+
+Qtile can also be run as a Wayland compositor rather than an X11 window
+manager. This does not require an X server or ``xcffib`` to be installed.
+Instead Qtile uses pywlroots_, a Python binding around the wlroots_ library,
+pywayland_ and python-xkbcommon_. First, wlroots needs to be installed, and
+must be the latest release. Be aware that some distributions package outdated
+versions of wlroots. The three Python libraries can then be installed using
+``pip install`` as above.
+
+Qtile can then be run directly from your TTY, specifying the backend:
```

Comment:
It might be good to note that this can also be run within a running X11 session, and will launch a nested window.

Reply:
Yeah good idea